demo/common/src/main/java/com/gentics/mesh/demo/AbstractDemoDumper.java

```diff
@@ -13,60 +13,53 @@
 import com.gentics.mesh.dagger.MeshComponent;
 import com.gentics.mesh.error.MeshSchemaException;
 import com.gentics.mesh.etc.config.MeshOptions;
-import com.gentics.mesh.etc.config.OrientDBMeshOptions;
 import com.gentics.mesh.util.UUIDUtil;
 
 /**
- * The demo dump generator is used to create a mesh database dump which contains the demo data. This dump is packaged and later placed within the final mesh jar
+ * The demo dumper is used to create a mesh database dump which contains the demo data. This dump is packaged and later placed within the final mesh jar
  * in order to accelerate demo startup.
  */
-public class DemoDumpGenerator {
-
-	static {
-		System.setProperty("memory.directMemory.preallocate", "false");
-		System.setProperty("vertx.httpServiceFactory.cacheDir", "data" + File.separator + "tmp");
-		System.setProperty("vertx.cacheDirBase", "data" + File.separator + "tmp");
-		if ("jotschi".equalsIgnoreCase(System.getProperty("user.name"))) {
-			System.setProperty("storage.wal.allowDirectIO", "false");
-		}
+public abstract class AbstractDemoDumper<T extends MeshOptions> extends AbstractMeshOptionsDemoContext<T> implements DemoDumper {
```

Comment:
DemoDumpGenerator name remained unchanged for the default end-implementation, so Dumper name is given for an abstraction.